lib/data-table/table.rb

```diff
@@ -252,10 +252,17 @@ def render_group_recursive(collection, index = 1, group_parent = nil, ancestors
     #############
     def render_totals
       html = '<tfoot>'
+
+      first_rendered_total = nil
+
       @total_calculations.each_with_index do |totals_row, index|
+        if first_rendered_total.nil? && !totals_row.empty?
+          first_rendered_total = 'first'
```

Comment:
won't this render `first` for all total rows, since the variable is defined outside of the loop block? The test case would be, if you have more than one total calculation, that they don't all get the css class.

Reply:
It will render first for all rows but not because the variable is defined outside of the loop block but because I never set `first_rendered_total` back to nil.

Reply:
Thanks for the test suggestion.  About to add that and then refactor what I have.

Reply:
Why you need to reset because you track this state over the life of the render_totals method, but why is that necessary? Isn't knowing whether or not a particular total row is the first total row knowable from entirely within this block?

Reply:
Yea, you are right.

Reply:
I just noticed Travis is running ci every time I push.

Reply:
@aharpervc - This is ready to be merged into it's parent branch.  Do you have any objections?